### PR TITLE
Jspp polars fix

### PR DESCRIPTION
**Pull request by jspaezp**
@cia23 

LMK if you want to add these guys :) noticed I had the dangling branch.
RN it is not ruff compliant but can be tidied up to be good to go.

### CODE REVIEW

File: gopher/annotations.py [hunk 1]
```diff
@@ -146,25 +154,48 @@ def load_annotations(species, aspect="all", release="current", fetch=False):
...
+    annot = polars_annot.collect().to_pandas()
 
-    keep = ["uniprot_accession", "go_id", "aspect"]
-    annot = annot.loc[:, keep].drop_duplicates()
-    annot["go_name"] = annot["go_id"].map(terms)
     return annot, mapping
+
+
+if __name__ == "__main__":
+    load_annotations("human", aspect="all", release="current", fetch=False)
```
Comment (cia23) [inline gopher/annotations.py:201]: I don't think we need this? We load annotations the first time a function requiring annotations is called.

**Review by cia23 — COMMENTED** [review]
Overall looks good to me! Just one comment. To get it to pass the linting, it looks like black just needs to be run on all the files.

File: gopher/annotations.py (hunk 1 above)
Comment (jspaezp) [reply gopher/annotations.py:201]: You are right, I just used this during testing locally and forgot totake it out :P

**Comment by jspaezp** [comment]
I think it is g2g!

**Pull request merged**